src/main/java/com/cinema_booking/booking/seat/SeatRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.cinema_booking.booking.seat;
+
+import java.util.UUID;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import com.cinema_booking.booking.seat.model.Seat;
+
+public interface SeatRepository extends JpaRepository<Seat, UUID> {
```

Comment:
Actually is it really necessary to store all seats in database? Considering the fact that we will have like 12 rooms in cinema, with 200 seats. That gives us 12 * 200 = 2400 seats. It won't be really optimal to go thorugh database every single time to find one.

Reply:
hmm the solution from article work but i left 
<path>
  <groupId>org.projectlombok</groupId>
  <artifactId>lombok-mapstruct-binding</artifactId>
  <version>0.2.0</version>
</path>

Reply:
I think that i should use Many to Many with additional column ex. isReserved: boolean it will be a good solution ?. I used actual solution bcs some seats can be premium but it  is also possible to make these feature in many to many relations